ui-tests/src/test/java/org/openmrs/reference/page/OpenConceptLabPage.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class OpenConceptLabPage extends Page {
+    
+    private static final By SUBSCRIPTION_URL = By.id("subscription-url");
+    private static final By SETUP_SUBSCRIPTION_BUTTON = By.cssSelector("#body-wrapperhome div:nth-child(2) div:nth-child(2) a");
+    private static final By CONCEPT_URL_FIELD = By.id("subscription-url");
+    private static final By TOKEN_URL_FIELD = By.id("subscription-token");
+    private static final By SAVE_BUTTON = By.className("saveSubscription");
+    private static final By CANCEL_BUTTON = By.cssSelector("#body-wrapper > subscription > div:nth-child(1) > form > fieldset > div:nth-child(4) > button:nth-child(2)");
+    private static final By EDIT_SUBSCRIPTION = By.cssSelector("#body-wrapper > home > div:nth-child(3) > div:nth-child(1) > a");
+    private static final By IMPORT_FROM_SUBSCRIPTION_BUTTON = By.cssSelector("#body-wrapper > home > div:nth-child(3) > div:nth-child(2) > fieldset > div:nth-child(2) > div > p:nth-child(1) > button");
+    private static final By CHOOSE_FILE = By.cssSelector("#body-wrapper > home > div:nth-child(3) > div:nth-child(2) > fieldset > div:nth-child(2) > form > p:nth-child(2) > input");
```

Comment:
For CHOOSE_FILE element refer to my comment [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/566/files#r713082619) !